Assets/Scripts/Models/Time/WorldTime.cs

```diff
@@ -292,15 +292,41 @@ public int Year
         return Math.Abs(time1.Seconds - time2.Seconds) > float.Epsilon;
     }
 
+    public override bool Equals(object obj)
+    {
+        if (obj.Equals(this))
+        {
+            return true;
+        }
+        else if (obj is WorldTime && ((WorldTime)obj) == this)
```

Comment:
Yes sure, it was more a redundancy thing to be purely explicit but I can change it if it really bugs you :P.